src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,29 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+type TodoProp = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<TodoProp> = ({ todo }) => {
+  const {
+    userId, title, completed, user,
+  } = todo;
+
+  return (
+    <li
+      className={classNames('TodoInfo', { 'TodoInfo--completed': completed })}
```

Comment:
It is better to keep it maximally clear to those who read
```suggestion
      className={classNames('TodoInfo', { 
        'TodoInfo--completed': completed 
      })}
```